packages/i18n/src/index.js

```diff
@@ -34,7 +35,9 @@ export function setLocaleData( localeData = { '': {} }, domain = 'default' ) {
 		} );
 	}
 
-	i18n.options.locale_data[ domain ] = localeData;
+	i18n.options.locale_data[ domain ] = i18n.options.locale_data.hasOwnProperty(domain)
```

Comment:
Code style†: Space between parentheses ([reference](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#arrays-and-function-calls)

† This repository is not (yet) enforced by ESLint standards.